src/api/built-in-components.md

```diff
@@ -101,6 +101,14 @@ Provides animated transition effects to a **single** element or component.
     <div v-if="ok">toggled content</div>
   </Transition>
   ```
+  
+  Transition on changing key attribute:
+  
+  ```vue-html
+  <Transition>
+    <div :key="data-id">{{changing-text}}</div>
```

Comment:
The values `data-id` and `changing-text` wouldn't work in practice, as the hyphens would be treated as subtraction operators.

Maybe something like this instead?

```suggestion
  Forcing a transition by changing the `key` attribute:
  
  ```vue-html
  <Transition>
    <div :key="text">{{ text }}</div>
```